build/Dockerfile

```
@@ -3,7 +3,7 @@ FROM registry.access.redhat.com/ubi8/ubi
 MAINTAINER OpenShift PSAP Team <openshift-psap@redhat.com>
 
 LABEL 	io.k8s.display-name="OpenShift PSAP Ci artifacts" \
-      	io.k8s.description="This are artifacts to run PSAP Ci" \
+      	io.k8s.description="This are artifacts to run PSAP CI" \
```

```suggestion
      	io.k8s.description="An image for running Ansible artifacts for OpenShift PSAP CI" \
```
I think this description could be reworded to be more meaningful and clear.